package.json

```diff
@@ -2,7 +2,7 @@
   "name": "immer",
   "version": "0.1.2",
   "description": "Create your next immutable state by mutating the current one",
-  "main": "immer.js",
+  "main": "immer-es5.js",
```

Comment:
main is now  `immer-es5.js`, so the unit tests are testing only immer-es5.js and not immer.js . Is this intended? or may be I am missing something.